app/controllers/posts_controller.rb

```diff
@@ -213,7 +213,13 @@ def copy_code
 
   def similar_to
     @post = Post.find(params[:id])
-    @posts = ENV["BONSAI_URL"] ? Post.includes(:user).search(@post.tags.presence || @post.title, 4).records.where.not(id: @post.id).select_overview_columns.public?.limit(3) : Post.where.not(id: @post.id).last(3)
+    @posts = if ENV["BONSAI_URL"] then
+        Rails.cache.fetch("#{cache_key_with_version}/similar_to", expires_in: 6.hours) do
```

Comment:
This is far too aggressive. Maybe something along the lines of 5 minutes?

Reply:
I seem to remember our caching doesn't work with expires_in anyway, since it's in memory. Wouldn't 5 minutes be far too short though, it wouldn't cache anything would it?

Reply:
It would prevent duplicate requests from being sent to Elasticsearch if a user refreshes the post's page.